<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Satellites</string>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>220</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Satellite position</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Range:</string>
    </property>
   </widget>
   <widget class="QLabel" name="range_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Elevation:</string>
    </property>
   </widget>
   <widget class="QLabel" name="el_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Azimuth:</string>
    </property>
   </widget>
   <widget class="QLabel" name="az_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Time and location</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Timezone:</string>
    </property>
   </widget>
   <widget class="QLabel" name="tz_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Lon:</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="lat_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Lat:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lon_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Satellite and orbital info:</string>
   </property>
   <widget class="QLabel" name="norad_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>NORAD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="sma_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Semi major axis:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ecc_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Eccentricity:</string>
    </property>
   </widget>
   <widget class="QLabel" name="incl_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Inclination:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ma_label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean anomaly:</string>
    </property>
   </widget>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>--</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
